### DISCUSSION TITLE
Projects Beta: "No Status" status / column disappearing act is confusing

### DISCUSSION THREAD

**Discussion by lsl** [community/community#5820] [Projects and Issues] [answered]
I like to have a "Triage" column stakeholders can place new cards or issues into for triage. Mostly I just use this as a place to shove notes and reminders that need circling back to. These get investigated and fleshed out before being scheduled into a "Todo" status / column.

The new "No Status" thing is confusing, mostly because the column disappears from the board view so its hard to know where to put new cards if there isn't anything currently in "No Status".

You might suggest I just create a Triage column and use that like before but this results in two clicks to add new issues to a project instead of one.

A better way to do this would be to let us set a default status for new the project issues. (I think the old project boards do this)

**Comment by azenMatt** [comment, marked as answer]
Thanks for the feedback @lsl - we have some automations launching _soon_ which includes a default landing status - lets see if that helps this once it is live.

**Reply by lsl** [reply]
That sounds perfect, thanks for the heads up!

**Reply by jer-k** [reply]
Is there a change log or release notes for updates on Projects Beta? We're just getting started and I would really like to be able to follow along!

**Reply by timoyates** [reply]
@jer-k Yep! There's a new changelog entry every two weeks or so.

**Reply by azenMatt** [reply]
Hi @jer-k - we just published this one yesterday for the new workflows https://github.blog/changelog/2021-09-29-the-new-github-issues-09-29-update/

@lsl - let me know if this helps with the default status values! 

Note - we are currently working on the live updates, so the need to refresh the page to see the result will be going away soon!

**Reply by lsl** [reply]
It does, thank you!

**Comment by elyobo** [comment]
Quite related, so putting it here (apologies if this should be a new issue), but we'd like to just use "no status" _as_ the "triage" column, but most UIs do not allow removing the status. You can do it in the board view by dragging it back, but you can't select no status or unselect a status on the table view or on the full issue pages - the inconsistency is a bit weird.

**Reply by lsl** [reply]
Not quite what you're asking for but with the workflows mentioned above you can set the default status/column when new things are added to the project.

![Screenshot 2021-10-01 at 17-52-24 Build software better, together](https://user-images.githubusercontent.com/811776/135584976-46fbd081-5259-4230-b7d9-4709e479239a.png)

**Reply by elyobo** [reply]
Yes thanks, that would be the alternative and would work for us as well - just a pain for all the issues that are already in there with no status :sweat_smile: And still leaves the "no status" UX a bit inconsistent.
